### PR TITLE
Enhancement By: Renaming 'ReadMes' to 'README' for Improved Clarity.

### PR DESCRIPTION
**Pull request by Rituraj-13**
**Fixes issue -** #42979 

**Made the required changes in README.md file .**

image

1. Updated `Localization ReadMes` to `Localization READMEs`.

### CODE REVIEW

**Review by aj11anuj — COMMENTED** [review]
`Localized README.md's` seems more appropriate to me than `Localization of README.md's`

Btw, I'm kind of getting some Hacktoberfest feel now 👀

**Comment by Rituraj-13** [comment]
> `Localized README.md's` seems more appropriate to me than `Localization of README.md's`
> 
> Btw, I'm kind of getting some Hacktoberfest feel now 👀

I find 'Localization README.md' to be inappropriate, and 'ReadMes' also require correction.

**Comment by aj11anuj** [comment]
Hello @Rituraj-13 brother, I appreciate your contribution, but maybe you got my words in wrong way. I mean `Localized README.md's` seems more appropriate to me not `Localization ReadMe` that was earlier. But `Localization README.md's` in some way of saying is also Ok-Ok. I don't know, if a minute change like this one will get accepted.

**Comment by Rituraj-13** [comment]
> Hello @Rituraj-13 brother, I appreciate your contribution, but maybe you got my words in wrong way. I mean `Localized README.md's` seems more appropriate to me not `Localization ReadMe` that was earlier. But `Localization README.md's` in some way of saying is also Ok-Ok. I don't know, if a minute change like this one will get accepted.

Thanks bro, for suggestions 👍
I made the required changes, have a look.

**Review by sandipanpanda — COMMENTED** [review]
/lgtm
Thanks!

File: README.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ If you need help at any point when contributing, the [New Contributor Ambassador
 | -------------------------- | -------------------------- | -------------------------- |
 | Arsh Sharma                | @arsh                      | @RinkiyaKeDad              |
 
-## Localization `README.md`'s
+## Localized `README.md`'s
```
Comment (windsonsea) [inline README.md:185]:
```suggestion
## Localized READMEs
```
Do we need to keep consistency in context?

Comment (Rituraj-13) [reply README.md:185]: Yeah, that would be better 👍

**Comment by windsonsea** [comment]
Seems you run different command series and /lgtm is removed.

/label tide/merge-method-squash

**Comment by Rituraj-13** [comment]
> Seems you run different command series and /lgtm is removed.
> 
> /label tide/merge-method-squash

Yeah, for some reason I couldn't make changes with those commands.
So I made a new commit.

So now, do I need to squash the 2 commits and again force push them ?

**Comment by nate-double-u** [comment]
I think "Localization" is likely correct in this instance.

"Localization ReadMes" could probably be corrected to "Localization READMEs", and I'd also suggest taking the "``" out of the title.

File: README.md [hunk 2]
```diff
@@ -182,7 +182,7 @@ If you need help at any point when contributing, the [New Contributor Ambassador
 | -------------------------- | -------------------------- | -------------------------- |
 | Arsh Sharma                | @arsh                      | @RinkiyaKeDad              |
 
-## Localization `README.md`'s
+## Localized READMEs
```
Comment (nate-double-u) [inline README.md:185]:
```suggestion
## Localization READMEs
```

File: README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@
 This repository contains the assets required to build the [Kubernetes website and documentation](https://kubernetes.io/). We're glad that you want to contribute!
 
 - [Contributing to the docs](#contributing-to-the-docs)
-- [Localization ReadMes](#localization-readmemds)
+- [Localized READMEs](#localized-readmemds)
```
Comment (nate-double-u) [inline README.md:8]:
```suggestion
- [Localization READMEs](#localization-readmes)
```

**Comment by nate-double-u** [comment]
/lgtm

If someone from the localization team agrees I think this can go in. 

Thinking about it more, and not being able to read the majority of the localizations, if the non-english README pages are just a translation of this page, then “Localized README” is probably correct, but if there are region/language specific bits of content that have been added to them, then the current “Localization READMEs” is probably correct.

But that’s a lot of thinking for a small change 😅 Probably this should go in, and if it needs to be updated again we should package it with a more robust review/edit of the page.

**Comment by sftim** [comment]
A better title could have been: Reword 'ReadMes' to 'READMEs'

**Pull request merged**